logging/Limiting_Access_to_Logs.md

```diff
@@ -122,23 +204,52 @@ and roles and between users and back-end roles.
 By assigning a single back-end role to a user, the user can 
 essentially be assigned to multiple roles at once.
 
-When you run the `onboard.sh` script to add Kibana tenant space, two new roles are created and role mappings are defined to link the new roles (and one pre-existing role) to a new back-end role.  The following table depicts the roles, back-end roles and role mappings that are created when the script is run to onboard the SAS Viya tenant `acme` from the `production` Viya namespace:
+### Roles and Role Mappings for a Tenant or Namespace
+
+When you run the `onboard.sh` script to add Kibana-tenant space, two new roles 
+are created and role mappings are defined to link the new roles (and one 
+pre-existing role) to a new back-end role.  The following table depicts the 
+roles, back-end roles and role mappings that are created when the script is 
+run to onboard the SAS Viya tenant `acme` from the `production` SAS Viya 
+namespace:
 
 | Back-end Role | Role | Purpose |
 | --- | --- | --- |
-|     | v4m_kibana_user | allows access to Kibana |
-| production_acme_kibana_users | search_index_production_acme | Allows access to log messages from the `production` namespace and `acme` tenant |
-|     | tenant_production_acme | Allows access to Kibana tenant space for `production_acme` |
+|     | v4m_kibana_user | Grants access to Kibana |
+|     | tenant_production_acme | Grants access to Kibana-tenant space for `production_acme` |
+| production_acme_kibana_users | search_index_production_acme | Grants access to log messages from the `production` namespace and `acme` tenant |
```

Comment:
_Grants access to log messages from the `acme` tenant within the `production` namespace_

Reply:
Revised: 
|     | search_index_production_acme | Grants access to log messages from the  `acme` tenant within the `production` namespace |